### ISSUE TITLE
Possibility to remove or hide counter on the title page

### ISSUE THREAD

**Issue by kellard** [bast/cicero#80]
I think the title page shouldn't have a counter, at least not a visible one. Could it be removed and start the counter on second slide or just hide it. It's a minor issue but for me the title page should be clean and focus on topic.

**Comment by bast** [comment]
I agree with that. I will see whether this can be tweaked but if yes, I agree that the title page should not show the slide count.
